dm/ctl/ctl.go

```diff
@@ -74,9 +65,37 @@ func Start(args []string) {
 		master.NewPurgeRelayCmd(),
 		master.NewMigrateRelayCmd(),
 	)
+}
 
-	rootCmd.SetArgs(args)
+// Init initializes dm-control
+func Init(cfg *common.Config) error {
+	// set the log level temporarily
+	log.SetLevel(zapcore.InfoLevel)
 
+	return errors.Trace(common.InitUtils(cfg))
+}
+
+// Usage prints usage
+func Usage() {
+	fmt.Println("Available Commands:")
+	for _, cmd := range rootCmd.Commands() {
+		fmt.Println("\t", cmd.Name())
```

Comment:
can we also print _short description_ here?

Reply:
sure, print [`cmd.Use`](https://github.com/pingcap/dm/pull/287/files#diff-8331c39cac1a8817f638729d331ae2dfR82)